tests/e2e/scripts/daiosmov16.json

```diff
@@ -1,6 +1,6 @@
 {
-    "weights": "5udai,5uosmo",
-    "initial-deposit": "5000000000udai,5000000000uosmo",
+    "weights": "5ibc/0CD3A0285E1341859B5E86B6AB7682F023D03E97607CCC1DC95706411D866DF7,5uosmo",
+    "initial-deposit": "5000000000ibc/0CD3A0285E1341859B5E86B6AB7682F023D03E97607CCC1DC95706411D866DF7,5000000000uosmo",
```

Comment:
I like that you made token0 DAI to ensure it translates to token1 in the upgrade (although it might just reorder lexicographically for balancer, im not sure of the top of my head)

Reply:
Our upgrade handler logic can handle any order of denoms in the balancer pool. There are unit tests confirming this